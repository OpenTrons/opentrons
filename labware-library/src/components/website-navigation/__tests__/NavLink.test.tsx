describe('NavLink', () => {
  it.todo('replace deprecated enzyme test')
})
