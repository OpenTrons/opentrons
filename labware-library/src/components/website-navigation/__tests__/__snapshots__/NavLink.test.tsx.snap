// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLink component renders 1`] = `
<div
  className="link_group"
>
  <a
    className="link_title"
    data-gtm-action="click"
    data-gtm-category="category"
    data-gtm-label="label"
    href="bar"
    rel="noopener noreferrer"
    target="_blank"
  >
    foo
  </a>
</div>
`;
